script.js

```diff
@@ -1,105 +1,305 @@
+            computerScoreDisplay.style.fontWeight = "bold";
+
+    const computerOptions = document.createElement("div");
+        document.body.appendChild(computerOptions);
+            computerOptions.style.display = "flex";
+            computerOptions.style.justifySelf = "stretch";
+            computerOptions.style.justifyContent = "center";
+            computerOptions.style.backgroundColor = "#DC9E82"
+
+
+    const computerRockPic = document.createElement("img");
+        computerRockPic.src = "rock-576669_1280.png";
+        computerOptions.appendChild(computerRockPic);
+        computerRockPic.style.width = "100px";
+        computerRockPic.style.height = "100px";
+        computerRockPic.style.margin = "50px";
+
+    const computerPaperPic = document.createElement("img");
+        computerPaperPic.src = "paper-23652_1280.png";
+        computerOptions.appendChild(computerPaperPic);
+        computerPaperPic.style.width = "100px";
+        computerPaperPic.style.height = "100px";
+        computerPaperPic.style.margin = "50px";
+
+    const computerScissorsPic = document.createElement("img");
+        computerScissorsPic.src = "scissors-2026687_1280.png";
+        computerOptions.appendChild(computerScissorsPic);
+        computerScissorsPic.style.width = "100px";
+        computerScissorsPic.style.height = "100px";
+        computerScissorsPic.style.margin = "50px";
+
+
+//create computer choice function 
 
 function getComputerChoice() {
     let randomValue = Math.floor(Math.random() * 3);
 
     if(randomValue === 0) {
+        computerRockPic.style.transform = "scale(1.5)";
+        computerPaperPic.style.transform = "scale(.5)";
+        computerScissorsPic.style.transform = "scale(.5)";
         return "rock";
 
     } else if(randomValue === 1) {
+        computerRockPic.style.transform = "scale(.5)";
+        computerPaperPic.style.transform = "scale(1.5)";
+        computerScissorsPic.style.transform = "scale(.5)";
         return "paper";
 
     } else {
+        computerRockPic.style.transform = "scale(.5)";
+        computerPaperPic.style.transform = "scale(.5)";
+        computerScissorsPic.style.transform = "scale(1.5)";
         return "scissors";
-    }
+    } 
 }
 
-//console.log(getComputerChoice(3));
+//create playRound function
 
-//function to get human choice with prompt
-// function to get human choice
-function getHumanChoice() {
-    let promptChoice = prompt("Rock, paper, or scissors?");
+function playRound(humanChoice, computerChoice) {
 
-        if(promptChoice.toLowerCase() === "rock") {
-            return "rock";
+    if (humanChoice === computerChoice) { 
+        winner.textContent = `It's a draw. Play again.`;
+    }
 
-        } else if(promptChoice.toLowerCase() === "paper") {
-            return "paper" 
+    else if (humanChoice === 'rock' && computerChoice === 'scissors' || humanChoice === 'paper' && computerChoice === 'rock' || humanChoice === 'scissors' && computerChoice === 'paper') {
```

Comment:
This logic is a little bit like a run on sentence. it might work but it doesn't look quite right. it may be helpful to give it more structure and separate each conditional a little. try something like:

` (humanChoice === "rock" && computerChoice === "scissors") ||
    (humanChoice === "paper" && computerChoice === "rock") ||
    (humanChoice === "scissors" && computerChoice === "paper")`